tools/migration.py

```diff
@@ -0,0 +1,248 @@
+from __future__ import absolute_import
```

Comment:
General remark: this script will stop working as soon as we have the branch-specific blacklists, right? please discuss with Martin if this is intended or if we need to support them